src/hb-ot-shape-complex-use-table.hh

```diff
@@ -29,7 +29,7 @@
  * # Not derivable
  * # Initial version based on Unicode 7.0 by Andrew Glass 2014-03-17
  * # Updated for Unicode 10.0 by Andrew Glass 2017-07-25
- * # Ammended for Unicode 10.0 by Andrew Glass 2018-09-21
+ * # Amended for Unicode 10.0 by Andrew Glass 2018-09-21
```

Comment:
This is a generated file, so the source need to be fixed. The source is https://github.com/harfbuzz/harfbuzz/blob/main/src/ms-use/IndicPositionalCategory-Additional.txt, but this is just a local copy and any changes to it will be eventually overridden. The upstream is https://github.com/microsoft/font-tools/blob/master/USE/IndicPositionalCategory-Additional.txt.